RELEASE.md

```diff
@@ -13,6 +13,10 @@ To create a new pre-release:
 7. After the release is published, another workflow should start automatically and update json files in https://github.com/paritytech/resolc-bin. Check the changes.
 8. Update the [contract-docs](https://github.com/paritytech/contract-docs/) accordingly
 
+# `resolc` NPM package release
+
+Will happen automatically. After the GH release process described above was successfull, ensure that the newly created release is marked as the latest release.
```

Comment:
> ensure that the newly created release is marked as the latest release.

this should not matter anymore,  the package will b e published with the tag version

Reply:
Right, I will remove it, thanks